packages/foundry/contracts/test/oracles/CrabOracleTest.t.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity >=0.8.13;
+
+import "forge-std/src/Test.sol";
+import "@yield-protocol/utils-v2/contracts/access/AccessControl.sol";
+import "../../oracles/crab/CrabOracle.sol";
+import {ICrabStrategy} from "../../oracles/crab/CrabOracle.sol";
+import "../../oracles/uniswap/UniswapV3Oracle.sol";
+
+contract CrabOracleTest is Test {
+    CrabOracle public crabOracle;
+    bytes6 baseId = 0x323900000000;
+    bytes6 quoteId = 0x303000000000;
+    UniswapV3Oracle uniswapV3Oracle =
+        UniswapV3Oracle(0x358538ea4F52Ac15C551f88C701696f6d9b38F3C);
+
+    function setUp() public {
+        crabOracle = new CrabOracle();
```

Comment:
Please follow the pattern in the other tests of using `vm.createSelectFork('mainnet', <block_number>)` for the first line in `setUp`.

This way it plays nice with the CI.